src/main/java/com/example/Car.java

```diff
@@ -0,0 +1,12 @@
+package com.example;
+
+public class Car {
+            private int speed;
+
+
+    public int getSpeed() {return speed;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 4, expected level should be 2.